### DISCUSSION TITLE
v0.8 user creation flow

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#900] [Q&A] [answered]
```ts
    client.collection('users').create({
      email,
      password,
      passwordConfirm: password
    })
```

The above seems to work, but do we now need to manually send verification emails via `client.collection('users').requestVerification(email)`?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, you have to manually trigger sending the verification email.
If you are authenticated as admin or your account has "Manage" access for the auth collection, you are allowed to directly set the `verified` state during create.
